frontend/app/src/utils/url.ts

```diff
@@ -55,15 +55,22 @@ export const getEtherScanRegisterUrl = (location: string) => {
         external: 'https://etherscan.io/register',
         route: { path: Routes.API_KEYS_EXTERNAL_SERVICES, hash: `#${location}` }
       };
-    case TRADE_LOCATION_POLYGON_POS:
+    // TODO: remove the string modification when https://github.com/rotki/rotki/issues/6725 is resolved
+    case TRADE_LOCATION_POLYGON_POS.split(' ').join('_'):
```

Comment:
how about this auto imported fn @lukicenturi 
```typescript
toSnakeCase(TRADE_LOCATION_POLYGON_POS)
toSnakeCase(TRADE_LOCATION_ARBITRUM_ONE)
```

Reply:
makes sense, let me try

Reply:
it works, updated.